js/algo.js

```diff
@@ -0,0 +1,84 @@
+function Flood_Fill_Algorithm(cell_id, target_color, replacement_color) {
```

Comment:
Good decision to create a new `algo.js` file. Keeps things more organized since you have added the `Bresenham_Line_Algorithm` function as well.